defaults/main.yml

```diff
@@ -28,28 +38,66 @@ php_scl_install_php: yes
 php_scl_php_packages:
   - "{{ php_scl_name }}"
   - "{{ php_scl_name }}-php-cli"
+  - "{{ php_scl_name }}-php-pdo"
+  - "{{ php_scl_name }}-php-mysql"
+  - "{{ php_scl_name }}-php-gd"
 
 # Should this role manage the configuration of PHP?
 #
 # yes | no
 php_scl_configure: yes
 
 # Timezone to set in the php.ini file
-php_scl_timezone: UTC
+php_scl_timezone: America/New_York
 
 # List of configuration files and options to insert
-#
+
+php_realpath_cache_size: 512k
+php_realpath_cache_ttl: 3600
+php_max_execution_time: 60
+php_max_input_vars: 2000
+php_memory_limit: 512M
+php_error_reporting: E_ALL & ~E_DEPRECATED
+php_html_errors: Off
+php_post_max_size: 10M
+php_upload_max_filesize: 10M
+php_default_socket_timeout: 300
+php_mysql_connect_timeout: 300
+php_upload_tmp_dir: "/var/tmp"
```

Comment:
These variables seem redundant when you can specify these below.

Reply:
Did you mean to remove all these from defaults ?
```
# List of configuration files and options to insert

php_realpath_cache_size: 512k
php_realpath_cache_ttl: 3600
php_max_execution_time: 60
php_max_input_vars: 2000
php_memory_limit: 512M
php_error_reporting: E_ALL & ~E_DEPRECATED
php_html_errors: Off
php_post_max_size: 10M
php_upload_max_filesize: 10M
php_default_socket_timeout: 300
php_mysql_connect_timeout: 300
php_upload_tmp_dir: "/var/tmp"
```

Reply:
yes, I think the defaults should be whatever the distribution's PHP package has the settings at. If removing the other values too, then these aren't used anyways.